pkg/controller/extension/actuator.go

```diff
@@ -55,6 +55,11 @@ func (a *actuator) Reconcile(ctx context.Context, log logr.Logger, ext *extensio
 		return fmt.Errorf("error reading Cluster object: %w", err)
 	}
 
+	if extensionscontroller.IsHibernationEnabled(cluster) {
```

Comment:
Isn't [`IsHibernated`](https://github.com/gardener/gardener/blob/282c229e4dc4b4088523b97a2a696a48e8506975/extensions/pkg/controller/shoot.go#L68) the more correct function to call here?

Reply:
I think with `IsHibernated()` the extension would error when the cluster is put to sleep and actually block it from reaching the hibernated status